utils/rpms/daos.spec

```diff
@@ -4,8 +4,8 @@
 %define daoshome %{_exec_prefix}/lib/%{name}
 
 Name:          daos
-Version:       0.6.0
-Release:       8%{?relval}%{?dist}
+Version:       0.6.1
+Release:       0%{?relval}%{?dist}
```

Comment:
New versions in RPMs typically have `Release: ` start at 1, not 0.

But if you are going to change the version here you should make sure the other places where the version are used are aligned.  `scons RELEASE=0.6.1 release` is typically used to do the work for you but it will create a separate PR for the new version.  Alternately you could do the work manually by updating [VERSION](https://github.com/daos-stack/daos/blob/a8645123797fa361266511a039369942ddafe65e/VERSION).

When you land this, with VERSION updated, it will create a new tag and a new release in Github.  You should probably confirm with @johannlombardi if this is something we really want to do.

Reply:
@johannlombardi do you want this to simply increment the release version?